README.md

```diff
@@ -12,17 +12,16 @@ By default, `kcp` only knows about:
 - [`CustomResourceDefinition`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s, to define new types
 - a handful of other low-level resources like [`Lease`](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/lease-v1/)s, [`Event`](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application-introspection/)s, etc.
 
-![kubectl api-resources showing minimal API resources](./docs/images/kubectl-api-resources.png)
+![kubectl api-resources showing kcp's API resources](./docs/images/kubectl-api-resources.png)
 
-Like vanilla Kubernetes, `kcp` persists these resources in etcd for durable storage.
+Any other resources, including standard Kubernetes resources like [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)s and the rest, can be added as CRDs and optionally reconciled using the standard controllers launched against the `kcp` API, or via two-way replication into a Kubernetes cluster.
 
-Any other resources, including Kubernetes-standard resources like [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/)s, [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)s and the rest, can be added as [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s and optionally reconciled using the standard controllers launched against the kcp API, or via two-way replication into a Kubernetes cluster.
 
 ## Why would I want that?
 
 Kubernetes is mainly known as a container orchestration platform today, but we believe it can be even more.
 
-With the power of [`CustomResourceDefinition`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s, Kubernetes provides a flexible platform for declarative APIs of _all types_, and the reconciliation pattern common to Kubernetes controllers is a powerful tool in building robust, expressive systems.
+With the power of CRDs, Kubernetes provides a flexible platform for declarative APIs of _all types_, and the reconciliation pattern common to Kubernetes controllers is a powerful tool in building robust, expressive systems.
```

Comment:
I think we need one more sentence in here.  This is a pitch for generic control plane (and the next three sentences).  You also need to pitch the key hook from today somewhere (i'm not sure exactly which line, but it's important):

> Imagine a declarative ... [unchanged]

> But a control plane without any workloads isn't very compelling - so we want to uplevel the majority of Kubernetes applications today by allowing them to move and spread across clusters without changing a line of YAML.  We believe moving a workload between clusters should be as easy as moving a pod between nodes.

> Take a multi-tenant control plane for everything, and add to it most of the world's cloud-native workloads happily cruising across clouds, datacenters, and out to the edge?

> That's **`kcp`**.

Reply:
Added to #576